justfile

```diff
@@ -98,7 +98,7 @@ download-data owner="ibis-project" repo="testing-data" rev="master":
 
 # start backends using docker compose; no arguments starts all backends
 up *backends:
-    docker compose up --wait {{ backends }}
+    docker compose up --build --wait {{ backends }}
```

Comment:
I don't think this change is necessary. I never have this problem with any of the containers we build, they get built automatically if necessary.

Reply:
Ah, nope. I'm wrong.

I don't really like this solution because it will rebuild images even if they don't need to be rebuilt, but I'm not sure what the alternative is.

Reply:
Docker layer cache will ensure that these are not actually rebuilt from scratch, it does nothing if the layer/image already exists.

Reply:
I see a build of the image every time no matter what when running just up postgres.

Reply:
I think it will still show the build no matter what, but the various steps should be prefixed with `CACHED`

Reply:
e.g.
```
 => [postgres 1/2] FROM docker.io/postgis/postgis:15-3.3-alpine@sha256:4738bee  0.0s
 => CACHED [postgres 2/2] RUN apk add --no-cache postgresql15-plpython3         0.0s
 ```

Reply:
Ah ok!